lib/tasks/editorials.rake

```diff
@@ -23,9 +23,9 @@ namespace :editorials do
   end
 
   desc "Send twice-weekly emails with possible submission rejections"
-  # Send on Monday and Thursday
+  # Send on Wednesday
```

Comment:
Let's talk about this tomorrow - I would like to not only do this once a week, just because of the backlog that could build up and be carried over from one AEiC to another.  At least, we should think about the schedule of this with respect to AEiC rotation

Reply:
Sorry, we forgot to talk about this today.

My rationale for picking Wednesday was that this would be a couple of days after the start of an EiC rotation so potentially the incoming EiC would have had a chance to review the new submissions and flag any that potentially might be out of scope.

One alternative might be to send this on Sunday at the _end_ of a rotation?

Reply:
Wednesday seems ok to start. Maybe we should try through a full AEIC cycle and see what people think of the timing?

Reply:
Right now, I feel like I own these decisions for submissions that come in while I am AEiC, and assuming I send out questions shortly after questionable submissions, I only can have 2 days after the end of my rotation that I need to pay attention.  If this goes out on Wednesdays, does that then mean that after my rotation is over, I have to pay attention for almost the full next week (W - F)?